src/services/launchtype/LaunchTypeService.ts

```diff
@@ -78,6 +78,10 @@ export class LaunchTypeService {
     }
   }
 
+  getMemoryUnit(): MemoryUnits {
+    return MemoryUnits.MB;
```

Comment:
I really like this solution, but I don't understand why the memory units will be influenced by the LanchType? As `convertBytes` always assumes that the input is bytes, we can inspect the passed value and determine the appropriate units to convert to.

Reply:
Hahaha, I did not know where else to put convertBytes. But how do you know appropriate units to convert to? Ideally, it would be user setting!

Reply:
So if `bytes >= (1024^1) && bytes < (1024^2)` = KB` etc

Reply:
Should I just put it in Util for now?

Reply:
Well if it's always going to be MB then there is no need to specify the Memory unit as part of the `convertBytes` function, instead just return `round((bytes / 1024 / 1024), 2) + " MB";` everytime. That being said, the code to figure out whether to treat as KB, MB, GB is trivial

Reply:
Ok, I would go with automatic version then. If bytes >= (1024^1) && bytes < (1024^2) use KB and so on. I'll do that!